fmanimashaun/budget-app
# Implement Splash Screen with Navigation to Sign In/Up

# Implement Splash Screen with Navigation to Sign In/Up

## Description:
In this PR, we have implemented a new splash screen for our application. The splash screen features a logo and two buttons for user authentication - "Sign In" and "Sign Up". Users can navigate to the respective sign-in or sign-up pages by clicking these buttons.

## Changes Made:
- Added a logo to the splash screen.
- Implemented "Sign In" and "Sign Up" buttons.
- Connected these buttons to navigate users to the respective Sign In or Sign Up pages.
- Written unit tests for both the controller and the view to ensure they render correctly and navigation works as expected.
- A system test has been added to confirm that the buttons navigate to the correct paths when clicked.

## How To Test:
1. Launch the application. You should be greeted with a splash screen displaying a logo and two buttons - "Sign In" and "Sign Up".
2. Click on the "Sign In" button. You should be navigated to the Sign In page.
3. Return to the splash screen and click on the "Sign Up" button. You should be navigated to the Sign Up page.
4. Run the unit tests for the controller and the view.
5. Run the system test to ensure that the buttons are working correctly.

## screenshots
![Screen Shot 2024-01-17 at 13 06 26](https://github.com/fmanimashaun/budget-app/assets/22054464/e9aed221-74a4-4a74-b950-eca3a599d302)
